### DISCUSSION TITLE
target file editor doesn't correctly resolve maven depedencies that are provided?

### DISCUSSION THREAD

**Discussion by jcompagner** [eclipse-pde/eclipse.pde#776] [General]
look at this picture:

![image](https://github.com/eclipse-pde/eclipse.pde/assets/3274268/c784cfaa-aeec-4624-a07d-9951ce8e0fae)

that should work right?

i include a repo that has some plugins
1 of those plugins depend on a version of antlr (i wanted that it is a range so i can update it easier in our final product target that you see above)

But for some reason the target editor always complains that it can't find antlr but it is there, in the mvn depedencies..

**Comment by merks** [comment]
If I recall correctly, the regular location cannot depend on things that are only available via the maven location because the maven location IUs are not available during resolution of those locations.  I think @laeubi improved that recently...

**Reply by laeubi** [reply]
This is an error from the Udate sitelocation that can only handle update-sites not other ones (including maven, directory, targlet ....) see 
- https://github.com/eclipse-pde/eclipse.pde/issues/207

Yes I planed working on this but didn't have time (yet).

**Comment by laeubi** [comment]
@jcompagner can you provide a minimal target file that reproduce your problem here:
- https://github.com/eclipse-pde/eclipse.pde/issues/207
